dev/tests/integration/testsuite/Magento/Framework/Image/Adapter/InterfaceTest.php

```diff
@@ -115,9 +115,12 @@ protected function _getFixture($pattern)
      */
     protected function _isFormatSupported($image, $adapter)
     {
+        if (!$image || !file_exists($image)) {
+            return false;
+        }
```

Comment:
Hello @anzin 
Could you please check my suggestion

```suggestion
        if ($image === null || !file_exists($image)) {
            return false;
        }
```

Reply:
Code fixed by recommendation, please check if every think is well.

Thank you.